src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -290,6 +292,23 @@ func create(ctx context.Context, s *service, r *taskAPI.CreateTaskRequest) (*con
 		return nil, err
 	}
 
+	// start network monitor
+	re := regexp.MustCompile("moby")
+	if re.MatchString(container.bundle) {
+		errCh := make(chan error, 1)
+		go func() {
```

Comment:
@bergwolf If I remember correctly, we use to have a config in the toml file, that could be used to enable to disable starting network monitor. This was useful for users to disable running another process to reduce memory usage when they network connect/disconnect was something not part of their use case. Now although netmon is not a separate process anymore, I still think it may be useful to have this config. Would like to hear your opinion on this. The config addition could be part of another PR to not block this one.

Reply:
In line 296, we have a check for `moby` containers because `netmon` is only required by moby. So I don't think we need an extra option to toggle it.